src/routes/transactions/helpers/swap-order.helper.spec.ts

```diff
@@ -165,7 +165,7 @@ describe('Swap Order Helper tests', () => {
     await expect(
       target.getOrder({
         chainId,
-        orderUid: orderUid as `0x${string}`,
+        orderUid: orderUid,
```

Comment:
Nit: we can now use the shorthand here:

```suggestion
        orderUid,
```

Reply:
This should be outdated also and overwritten by merges.